### PR TITLE
[FIX] *: remove references to `kanban_label_selection` widget

### PR DESCRIPTION
**Pull request by mgeubelle**

### CODE REVIEW

File: addons/web/static/src/js/fields/field_registry.js [hunk 1]
```diff
@@ -41,6 +41,7 @@ registry
     .add('attachment_image', basic_fields.AttachmentImage)
     .add('image_selection', basic_fields.ImageSelection)
     .add('label_selection', basic_fields.LabelSelection)
+    .add('kanban_label_selection', basic_fields.LabelSelection)
```
Comment (aab-odoo) [inline addons/web/static/src/js/fields/field_registry.js]: i'd add a `// deprecated, please use 'label_selection' instead` comment
(and i'd do the same for state_selection)

**Comment by aab-odoo** [comment]
robodoo rebase-ff

File: addons/web/static/src/js/fields/field_registry.js (hunk 1 above)
Comment (mge-odoo) [reply addons/web/static/src/js/fields/field_registry.js]: done

**Comment by aab-odoo** [comment]
robodoo r+

**Pull request closed without merging**